src/tpl/login.tpl

```diff
@@ -26,8 +27,8 @@
             border: 1px solid #AAA;
             padding: 20px;
             background-color: #f5f5f5;
-            width: 300px;
-            margin: 0 auto;
+            max-width: 300px;
+            margin: 20px auto;
         }
```

Comment:
undo the changes you made to .login-box here, and add 
```
@media (max-width:767px) {
	.login-box {
		width:80%;
	}
	.home-wrapper img.logo {
    	width:80%;
	}
}
```
to the bottom of the css